src/controllers/auth.controller.js

```diff
@@ -0,0 +1,41 @@
+import Response from '../helpers/response';
+import Password from '../helpers/password';
+import Token from '../helpers/token';
+import Userservices from '../database/services/userservice';
+
+class Authcontroll {
+    static async signUp(req, res) {
+        try {
+            const hashPassword = Password.hashPassword(req.body.password);
+            const { username, email } = req.body;
+            const userData = { username, email, password: hashPassword };
+            const user = await Userservices.createUser(userData);
+            const token = Token.generateToken(user.user_id);
+            const { user_id } = user.dataValues;
+            return Response.handleSuccess(201, 'Successful registered', { user_id, username, email, token }, res);
```

Comment:
Define all status codes in a constant file to be called here.
E.g:
 ```
  const HTTP_OK=200;
  const HTTP_CREATED=201;
  const HTTP_FORBIDDEN=403;
```

------------------------------------------------

Then call it like this:
      ` Response.handleSuccess(HTTP_CREATED,'Successful registered',........)`

etc...